easybuild/easyconfigs/c/CRISPResso2/CRISPResso2-2.1.2-foss-2020b-Python-2.7.18.eb

```diff
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonPackage'
+
+name = 'CRISPResso2'
+version = '2.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pinellolab/CRISPResso2/'
+description = """CRISPResso2 is a software pipeline designed to enable rapid and
+intuitive interpretation of genome editing experiments."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+sources = [
+    {'source_urls': ['https://github.com/pinellolab/CRISPResso2/archive/'],
+     'filename': 'v%(version)s.tar.gz'},
+    {'source_urls': ['https://crispresso.pinellolab.partners.org/static/demo/'],
+     'filename': 'base_editor.fastq.gz', 'extract_cmd': 'cp %s %(builddir)s'}]
```

Comment:
Should we rename this on download to include a datestamp, in case it gets updated in place upstream?
See `download_file` in https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

Reply:
changed...though, i thought checksums are enough to secure this, is there any particular reason for this?